src/server_manager/ui_components/outline-server-settings.html

```diff
@@ -19,11 +19,14 @@
 <link rel='import' href='../bower_components/paper-checkbox/paper-checkbox.html'>
 
 <link rel='import' href='./cloud-install-styles.html'>
+<link rel='import' href='./outline-server-settings-styles.html'>
```

Comment:
I don't see the file, is it commited? I'd prefer that we keep the styles in this file, unless it's being used by outline-validated-input?

Reply:
Fixed, not sure how that happened